src/clientsettingsdlgbase.ui

```diff
@@ -482,6 +489,25 @@
           <item>
            <widget class="CLanguageComboBox" name="cbxLanguage"/>
           </item>
+          <item>
```

Comment:
You need to add a tabstop (around line 724).

Reply:
I have rebased my code, now including #1415. In the new tabbed settings, a tabstop seems not to be required any more.